include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -19,6 +19,13 @@
 
 #include <CL/sycl.hpp>
 
+// Macros to check the new SYCL features
+#if defined(__LIBSYCL_MAJOR_VERSION) && defined(__LIBSYCL_MINOR_VERSION)
+#    define _ONEDPL_KERNEL_BUNDLE_PRESENT (__LIBSYCL_MAJOR_VERSION >= 5 && __LIBSYCL_MINOR_VERSION >= 2)
```

Comment:
It seems get_kernel_bundle was fixed on July 2nd ([commit](https://github.com/intel/llvm/commit/69a68a6dc16a537a1e40ff3cdd969b45b5e64037)).
Let's use ```__LIBSYCL_MINOR_VERSION >= 3```, because 2 was set in April 13th ([commit](https://github.com/intel/llvm/commit/6a49170027fba61cea649ffdd299ed095b5e3d75)), when get_kernel_bundle was not available. 3 was set on July 8th ([commit](https://github.com/intel/llvm/commit/962909fe9e78bd597594d67c5c5be7eacf8fb10a)). 

Also, I suggest waiting for #298 to merge and then using ```__LIBSYCL_VERSION```